novawallet/Modules/AssetList/Models/AssetListModelHelpers.swift

```diff
@@ -19,23 +19,24 @@ enum AssetListModelHelpers {
         from chain: ChainModel,
         assets: [AssetListAssetModel]
     ) -> AssetListGroupModel {
-        let value: Decimal = assets.reduce(0) { result, asset in
-            result + (asset.totalValue ?? 0)
+        let amountValue: AmountPair<BigUInt, Decimal> = assets.reduce(.init(amount: 0, value: 0)) { result, asset in
```

Comment:
This this changes sorting on all screens that contains assets list. As I understand in terms of the task we need only check sorting on the Swap Pay/Receive screens.

Reply:
same sort used in android app for common list